packages/npms.sls

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+{% from "packages/map.jinja" import packages with context %}
+
+{% set req_states = packages.npms.required.states %}
+{% set req_pkgs = packages.npms.required.pkgs %}
+{% set wanted_npms = packages.npms.wanted %}
+{% set unwanted_npms = packages.npms.unwanted %}
+
+### REQ PKGS (without these, some of the WANTED PIPS will fail to install)
+npm_req_pkgs:
+  pkg.installed:
+    - pkgs: {{ req_pkgs | json }}
+
+### NPM PKGS to install using npm
+# (requires the npm deb/rpm installed, either by the system or listed in
+# the required packages
+{% for nn in wanted_npms %}
+{{ nn }}:
+  npm.installed:
```

Comment:
From the same reference as above, we've got `dir`, which controls whether the installation is global or not.  Can this be incorporated here?  If so, will probably require `user` as well.  That may be beyond the scope of this PR.  However, it would be far easier to structure the pillar and states at this stage, than making (breaking) changes later on to accommodate for non-global installations.